docs/configuration.md

```diff
@@ -5,16 +5,19 @@ For non-core options (for example UI options), take a look at [Settings](https:/
 
 ### Base options
 
-|Option|Description|Default|
-|---|---|---|
-|`APP_NAME`|The gallery name|`Lychee`|
-|`APP_ENV`|Environment of your gallery, `production` or `development`|`production`|
-|`APP_URL`|The URL of your gallery (which resolves to the `public/` folder)|`http://localhost`|
-|`APP_KEY`|Your app key which is used for encryption (set during installation)|`null`|
-|`TIMEZONE`|The timezone of your photos, requires a named timezone identifier like `Europe/Paris`, don't use an offset (`+01:00`) or an abbreviation (`CEST`)|system timezone of server|
-|`LYCHEE_UPLOADS`|Path to uploads directory|`uploads/` inside `public/`|
-|`LYCHEE_UPLOADS_URL`|URL to uploads directory|`uploads/`|
-|`TRUSTED_PROXIES`|Trusted proxy IP addresses|`null`|
+| Option     | Description                                                                                                     | Default            |
+|------------|-----------------------------------------------------------------------------------------------------------------|--------------------|
+| `APP_NAME` | The gallery name                                                                                                | `Lychee`           |
+| `APP_ENV`  | Environment of your gallery, `production` or `development`                                                      | `production`       |
+| `APP_URL`  | The URL of your gallery (which resolves to the `public/` folder)                                                | `http://localhost` |
+| `APP_DIR`  | The URL path after the domain name of your gallery (which resolves to the `public/` folder), better left empty. | _empty_            |
```

Comment:
Wait, I seem to remember from preceding PRs that `APP_URL` is to hold URL up to the host name only, and the rest (if non-empty) goes in `APP_DIR`, correct?

If so, then the current description of `APP_URL` is confusing (clearly it's _not_ "The URL" if it's not a complete URL in some cases?!). Come to think of it, so is its name, but I can imagine that changing the latter may not be possible, or at least not worth the effort...

If what I wrote above is correct, I would change this to something like:

```suggestion
| `APP_URL`  | The URL of your gallery (which resolves to the `public/` folder). For installations hosted in a subfolder this should contain the part of the URL up to and including the host name only. | `http://localhost` |
| `APP_DIR`  | For installations hosted in a subfolder this should contain the rest of the URL (after the host name). | _empty_            |
```